services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -10,6 +10,8 @@ import { FormProvider, useForm } from "react-hook-form";
 import { FormPageTemplate } from "types/report";
 
 export const ReportPageWrapper = () => {
+  const praStatement =
+    "Under the Privacy Act of 1974 any personally identifying information obtained will be kept private to the extent of the law. According to the Paperwork Reduction Act of 1995, no persons are required to respond to a collection of information unless it displays a valid OMB control number. The valid OMB control number for this information collection is 0938-1053. The time required to complete this information collection is estimated to average 2.5 hours per response, including the time to review instructions, search existing data resources, gather the data needed, and complete and review the information collection. If you have comments concerning the accuracy of the time estimate(s) or suggestions for improving this form, please write to: CMS, 7500 Security Boulevard, Attn: PRA Reports Clearance Officer, Mail Stop C4-26-05, Baltimore, Maryland 21244-1850";
```

Comment:
is there any way we could move this to the general information content and access it from there? I think the PRA statement only exists for the General Information Page anyway, while the ReportPageWrapper is used generally for all report pages

Reply:
Oh I was overthinking of how to make that work but yes I can move this!

Reply:
@ajaitasaini Moved! ✅